src/Compilers/CSharp/Test/Semantic/Semantics/RefFieldTests.cs

```diff
@@ -12592,5 +12592,198 @@ public void ScopedReserved_Alias_Escaped()
                 Diagnostic(ErrorCode.HDN_UnusedUsingDirective, "using @scoped = System.Int32;").WithLocation(1, 1)
                 );
         }
+
+        [Fact, WorkItem(63018, "https://github.com/dotnet/roslyn/issues/63018")]
+        public void InitRefField_AutoDefault()
```

Comment:
>InitRefField_AutoDefault() [](http://example.com/codeflow?start=20&length=26)

Consider moving above `ScopedReserved_*` tests, to avoid a conflict with [https://github.com/dotnet/roslyn/pull/62783](https://github.com/dotnet/roslyn/pull/62783).